scripts/konflux/version.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
```

Comment:
[also consistency nit]
```suggestion
#!/usr/bin/env bash
```